### PR TITLE
EXPERIMENTAL: Stimulant symptom change

### PR DESCRIPTION
**Pull request by ghost**
### Intent of your Pull Request

BEFORE: Stimulant,  no downside symptom that gives a massive speedboost

AFTER: Stimulant, provides massive speedboost, but causes toxin damage over time. Helps negate cold as well. May end badly if plantperson/have a fever.
#### Changelog

:cl:
rscadd: EXPERIMENT-Stimulant now adds less of it's original chem, but adds some coffee and a variety of adrenalin that causes some light toxin damage. If it's too much, it'll be reverted and changed.
/:cl:

### CODE REVIEW

**Comment by BOBsHERE** [comment]
So now stimulant will be  fairly useless without toxin filter, which you are also nerfing.

File: code/datums/diseases/advance/symptoms/stimulant.dm [hunk 1]
```diff
@@ -30,9 +30,13 @@ Bonus
...
 		switch(A.stage)
 			if(5)
-				if (M.reagents.get_reagent_amount("ephedrine") < 10)
-					M.reagents.add_reagent("ephedrine", 10)
+				if (M.reagents.get_reagent_amount("ephedrine") < 5)
+					M.reagents.add_reagent("ephedrine", 5)
+				if (M.reagents.get_reagent_amount("coffee") < 5)
+					M.reagents.add.reagent("coffee", 5)
+				if (M.reagents.get_reagent_amount("Adrenaline") < 2)
+					M.reagents.add.reagent("Adrenaline", 2)
```
Comment (ShadowDeath6) [inline code/datums/diseases/advance/symptoms/stimulant.dm]: fuck no this is meth but on steroids with no brainloss, this is what fucking lings get speed boosted with for 2 seconds when they use adrenals and you're giving it to them permanently fuck no

Comment (ghost) [reply code/datums/diseases/advance/symptoms/stimulant.dm]: they also get a fuckload of tox damage if you forget

Comment (ShadowDeath6) [reply code/datums/diseases/advance/symptoms/stimulant.dm]: nevermind, this is just the stun reduction one, carry on.

also it only damages if you OD, which is 30

Comment (ShadowDeath6) [reply code/datums/diseases/advance/symptoms/stimulant.dm]: also also you need to set the reagent to "changelingAdrenaline"

**Comment by oisin100** [comment]
May as well remove Virology while you're at it.

The only issue i have with the stimulant virus right now is the fact that you can't arrest people using it without lethals, Maybe creating a custom chemical which doesn't make you stun resistant would be a better idea than killing virology.

**Comment by Super3222** [comment]
> Maybe creating a custom chemical which doesn't make you stun resistant would be a better idea than killing virology.

Bingo.

**Comment by ghost** [comment]
I would if i could, super. I know exactly what chem i'd want and everything. Issue is i cannot make a pr more than one file big, hence these clogs of prs everywhere, simply because my fucking norton antivirus is recognizing everything as a virus and not letting me install anything, so the only way for me to code is the online editor, and the only way I can use that is on my shitty iphone because I can't even access the github webpage at this point

So i would make a nice custom chemical that gives you a speedboost with no stun resistance and has a slight backlash but that would require me making multiple pr's for it so until I can muster the energy to make 3 prs for this one pr on my goddamn cracked iphone 3G this is the best you're getting

**Comment by BOBsHERE** [comment]
who says you cant arrest someone with stimulant? what? stun them once, get closer, stun them again then cuff. dont stun them once and try to walk up and cuff them. 

edit: im actually fine with removing stun resistance. Dont make it poison you though. Pick one.

**Pull request closed without merging**